shell/plugins/dashboard-store/__tests__/actions.spec.js

```diff
@@ -0,0 +1,125 @@
+import _actions from '@shell/plugins/dashboard-store/actions';
+
+const { findAll } = _actions;
+
+describe('dashboard-store: actions', () => {
+  let dispatchCalls = {};
+  let commitCalls = {};
+  // we're dumping our commits into a map so we can reference them
+  // actions and mutations are stored against their name in an array so we can capture if they're called multiple times
+  // arguments past the name of the action or mutation are stored in an array in case there are multiple
+  const commit = (mutation, ...args) => {
+    commitCalls[mutation] = [...(commitCalls[mutation] || []), args];
+  };
+  const dispatch = (action, ...args) => {
+    dispatchCalls[action] = [...(dispatchCalls[action] || []), args];
+    if (action === 'request') {
```

Comment:
No logic in tests. I am actually not sure why we do not have a linting rule here, but to add a condition in a test makes it worthless. Tests have to be deterministic and driven by data, mocks and stubs.

Reply:
removed the logic from the test, instead using the internal logic from jest's mocks to assert calls and params.